package.json

```diff
@@ -10,13 +10,14 @@
     "mixmax-git-hooks": "./dist/bin/index.js"
   },
   "scripts": {
-    "build": "del dist && rollup \"-c$([ \"$WATCH\" = true ] && echo w)\" && chmod +x dist/bin/index.js",
+    "build": "rm -rf dist/ && tsc",
```

Comment:
I think it's still better to use `del-cli` dependency for cross-platform compatibility

Reply:
We never work on non-unix platforms, but this is a public repo, so you're definitely right. Fixed.